src/renderer/tvgShape.h

```diff
@@ -319,9 +320,19 @@ struct Shape::Impl : Paint::Impl
                 rs.stroke->dashPattern = static_cast<float*>(malloc(sizeof(float) * cnt));
                 if (!rs.stroke->dashPattern) return Result::FailedAllocation;
             }
+            auto len = 0.0f;
             for (uint32_t i = 0; i < cnt; ++i) {
+                if (pattern[i] < 0.0f) {
```

Comment:
this check has to be done somewhere, if not here, then in the svg/lottie loaders.

Reply:
I think the engine is not obligated to handle those exceptions. Could you please address the reason?

Reply:
please check my comment below